administrator/components/com_admin/sql/updates/postgresql/4.1.0-2021-06-08.sql

```diff
@@ -0,0 +1,26 @@
+--
+-- Table structure for table "#__cookiemanager_cookies"
+--
+
+CREATE TABLE IF NOT EXISTS "#__cookiemanager_cookies" (
+  "id" serial NOT NULL PRIMARY KEY,
```

Comment:
```suggestion
  "id" serial NOT NULL,
```
When doing stuff for PostgreSQL and not being familiar with that kind of database, you should have a look at other create table statements for that kind of database.

Reply:
I have not committed directly from github, because it creates unnecessary 4 commits